react-client/src/utils/domUtils.js

```diff
@@ -89,11 +89,21 @@ function hasAnnotations(annotationStore, captureElement){
   return hasExistingAnnotations
 }
 
+// to check if the element has already an existing dialog box
+function hasDialogBox(dataId) {
+  const existingDialog = document.getElementById(`${dataId}`)
+  if (existingDialog !== null) {
```

Comment:
Just optional (not necessary to be pushed). But you can also return the boolean like this instead of using if-else:

`return existingDialog !== null`

Will still approve the PR.

Reply:
ah cge will implmement recommendations